addon/gocode/src/OSAG/goDB/liblz4_linux.go

```diff
@@ -1,3 +1,5 @@
+// +build linux
```

Comment:
These are redundant, as Go will pick this up from the _linux/_darwin by convention...

Reply:
Didn't know that. Perfect!